### ISSUE TITLE
For compilations, only the album artist is displayed for the tracks

### ISSUE THREAD

**Issue by owstr** [avdept/JellyBoxPlayer#100]
As you can see in the screenshots, Jellybox always shows “Various Artists”, while Jellyfin shows the respective artist via the web interface. Do I have an error in the mp3 tags or is this a bug?

<img width="1552" alt="Bildschirmfoto 2024-12-12 um 16 50 21" src="https://github.com/user-attachments/assets/6ce720d8-3262-46c1-a782-48cf59910f69" />
<img width="592" alt="Bildschirmfoto 2024-12-12 um 16 52 39" src="https://github.com/user-attachments/assets/14387b97-a7a6-4a73-bb2f-380e124e4e14" />
<img width="1336" alt="Bildschirmfoto 2024-12-12 um 17 17 03" src="https://github.com/user-attachments/assets/99b30387-c0a4-4a22-997e-7b6d84656620" />

**Comment by avdept** [comment]
I'm taking value from Album Artist, not Artist. Thats where it states Various Artists

**Comment by owstr** [comment]
Yes, obviously, but I think that's wrong for a compilation with different artists.

**Comment by avdept** [comment]
> Yes, obviously, but I think that's wrong for a compilation with different artists.

Yeah that makes sense. I'll see if I can change it, thanks for reporting!

**Comment by owstr** [comment]
With pleasure. And thanks for the quick response.

**Comment by avdept** [comment]
@owstr sorry for being this long to reply. This was fixed long ago, but I just totally forgot to reply
